Services/Certificate/classes/Placeholder/Description/class.ilCoursePlaceholderDescription.php

```diff
@@ -57,7 +57,7 @@ public function __construct(
         }
         $this->customUserFieldsPlaceholderDescriptionObject = $customUserFieldsPlaceholderDescriptionObject;
 
-        $customUserFieldsPlaceholderHtmlDescription = $this->customUserFieldsPlaceHolderDescriptionObject->getPlaceholderDescriptions();
+        $customUserFieldsPlaceholderHtmlDescription = $this->customUserFieldsPlaceholderDescriptionObject->getPlaceholderDescriptions();
```

Comment:
@M-Barz This is a strange one. I cannot spot the difference, but this 'change' was essential to get the test succeeding.